frontend/app/src/store/main.ts

```diff
@@ -1670,6 +1671,25 @@ export class MainStore {
       return false;
     }
   }
+
+  async  updateBountyPaymentStatus(created: number): Promise<any> {
+    try {
+      if (!uiStore.meInfo) return null;
+      const info = uiStore.meInfo;
+      const r: any = await fetch(`${TribesURL}/bounty/paymentstatus/${created}`, {
+        method: 'POST',
+        mode: 'cors',
+        headers: {
+          'x-jwt': info.jwt,
+          'Content-Type': 'application/json'
+        }
+      });
+
+      return r;
+    } catch (e) {
+      return false;
```

Comment:
lets log out the error here since this is a failure

Reply:
Okay